frontend/src/components/widgets/Button/Button.test.tsx

```diff
@@ -0,0 +1,110 @@
+/**
+ * @license
+ * Copyright 2018-2019 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react"
+import { fromJS } from "immutable"
+import { shallow } from "enzyme"
+import { buttonOverrides } from "lib/widgetTheme"
+import { WidgetStateManager } from "lib/WidgetStateManager"
+
+import { Button as UIButton } from "baseui/button"
+
+import Button, { Props } from "./Button"
+
+jest.mock("lib/WidgetStateManager")
+
+const sendBackMsg = jest.fn()
+
+const getProps = (elementProps: object = {}): Props => ({
+  element: fromJS({
+    id: 1,
+    label: "Label",
+    ...elementProps,
+  }),
+  width: 0,
+  disabled: false,
+  widgetMgr: new WidgetStateManager(sendBackMsg),
+})
+
+describe("Button", () => {
+  it("renders without crashing", () => {
+    const props = getProps()
+    const wrapper = shallow(<Button {...props} />)
+
+    expect(wrapper).toBeDefined()
+  })
+
+  it("Should have correct className and style", () => {
```

Comment:
I would propose to start the statements with the lowercase.

Reply:
agree 👍